ui/shared/status/StatusGifPopup.qml

```diff
@@ -130,7 +218,7 @@ Popup {
             SVGImage {
                 id: gifImage
                 anchors.horizontalCenter: parent.horizontalCenter
-                source: "./assets/img/gif.svg"
+                source: Style.current.name === "light" ? "./assets/img/gif.svg": "./assets/img/gif-dark.svg"
```

Comment:
```suggestion
                source: Style.current.name === Constants.lightThemeName ? "./assets/img/gif.svg": "./assets/img/gif-dark.svg"
```

Another option could be:
```
source: `./assets/img/gif-{Style.current.name}.svg`
```
and rename `gif.svg` to `gif-light.svg`

Reply:
Went for the second option, that is much nicer, thanks ;)